File: @LV1/1.8/dasblinkenlights/dasblinkenlights.rb
def gcd(a, b)
  return a if b.zero?

  gcd(b, a % b)
end

a, b, tl = gets.split.map(&:to_i)

lcm = a * b / gcd(a, b)

puts tl / lcm >= 1 ? 'yes' : 'no'

File: LV1/1.8/dasblinkenlights/dasblinkenlights.py
def gcd(a, b):
    if b == 0:
        return a
    return gcd(b, a % b)


a, b, tl = [int(i) for i in input().split(' ')]

lcm = a*b/gcd(a, b)

if tl/lcm >= 1:
    print("yes")
else:
    print("no")
